docs/modules/ROOT/pages/cdc-cassandra-events.adoc

```diff
@@ -0,0 +1,188 @@
+= CDC for Cassandra Events 
+
+The Datastax CDC for Cassandra agent pushes the mutation primary key for the CDC-enabled table into the Apache Pulsar events topic (also called the dirty topic). The messages in the data topic (or clean topic) are keyed messages where both the key and the payload are https://avro.apache.org/docs/current/spec.html#schema_record[AVRO records]:
```

Comment:
Could we please globally change all the `Datastax` to `DataStax` (notice the upper case `S`) to preserve our branding! Thanks!